### PR TITLE
freezetimers: fix concurrent modification exception

### PR DESCRIPTION
**Pull request by Netami1**
Fixes CME as listed in #1305

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java [hunk 1]
```diff
@@ -23,10 +23,7 @@
  */
 package net.runelite.client.plugins.freezetimers;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java]: Avoid wildcard imports

File: runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java [hunk 2]
```diff
@@ -101,14 +98,20 @@ public long getTimerReApply(Actor actor, TimerType type)
 
 	public boolean areAllTimersZero(Actor actor)
 	{
-		for (TimerType type : TimerType.values())
+		Iterator typeIterator = timerMap.keySet().iterator();
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java]: final

File: runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java [hunk 3]
```diff
@@ -101,14 +98,20 @@ public long getTimerReApply(Actor actor, TimerType type)
...
+
+		while(typeIterator.hasNext())
 		{
-			if (getTimerReApply(actor, type) > System.currentTimeMillis())
+			TimerType type = (TimerType) typeIterator.next();
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java]: final

File: runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java (hunk 2 above)
Comment (Lucwousin) [reply runelite-client/src/main/java/net/runelite/client/plugins/freezetimers/Timers.java]: Could also just make it a Iterator<TimerType>, so you don't have to do (TimerType)

**Pull request merged**